alica_engine/src/engine/PlanBase.cpp

```diff
@@ -397,6 +398,7 @@ void PlanBase::setLoopInterval(AlicaTime loopInterval)
 
 std::condition_variable* PlanBase::getStepModeCV()
 {
+    _stepCalled = true;
```

Comment:
this is confusing. Can you instead add a `stepNotify` method to plan base? AlicaEngine::stepNotify() will simply call planBase::stepNotify() & the implementation will be moved to plan base